docs/qcoverage/index.md

```diff
@@ -44,68 +44,91 @@ there is always one thread in addition to the number specified with
+* To do a build of qcoverage, first clone the adamajava repository.
+  ~~~~{.text}
+  git clone https://github.com/AdamaJava/adamajava
+  ~~~~
 
-Download the [qcoverage tar file](http://sourceforge.net/projects/adamajava/files/qcoverage.tar.bz2/download)
-Untar the tar file into a directory of your choice
+*  Then move into the adamajava folder:
+  ~~~~{.text}
+  cd adamajava
+  ~~~~
 
-You should see jar files for qcoverage and its dependencies:
+*  Run gradle to build qcoverage and its dependent jar files:
+  ~~~~{.text}
+  ./gradlew :qcoverage:build
+  ~~~~
+  This creates the qcoverage jar file along with dependent jars in the `qcoverage/build/flat` folder
 
-~~~~{.text}
-[oholmes@minion0 qcoverage]$ tar xjvf qcoverage.tar.bz2
-x antlr-3.2.jar
-x jopt-simple-3.2.jar
-x picard-1.110.jar
-x qbamfilter-1.0pre.jar
-x qcommon-0.1pre.jar
-x qpicard-0.1pre.jar
-X qio-0.1pre.jar
-x qcoverage-0.7pre.jar
-x sam-1.110.jar
-[oholmes@minion0 qcoverage]$
-~~~~
 
 ## Usage
 
-For sequence read depth:
-
-~~~~{.text}
-java -jar qcoverage.jar -t seq \
-    --bam test1.bam --gff3 GRCh37.gff3 -o report.vcf --log logfile
+~~~~
+usage: java -jar qcoverage.jar --type <type of coverage>  --input-bam <bam file> --input-gff3 <gff3 file> --output <output prefix> --log <log file> [options]
+Option              Description
+------              -----------
+--help              Show usage and help.
+--input-bai         Opt, a BAI index file for the BAM file. Def=<input-bam>.bai.
+--input-bam         Req, a BAM input file containing the reads.
+--input-gff3        Req, a GFF3 input file defining the features.
+--log               Req, log file.
+--loglevel          Opt, logging level [INFO,DEBUG], Def=INFO.
+--output            Req, the output file path. Here, filename extension will
+                      automatically added.
+--output-format     Opt, specify output file format, multi values are allowed.
+                      Possible values: [VCF, TXT, XML]. Def=TXT.
+--per-feature       Opt, to run the per-feature coverage mode. Default is to run
+                      standard coverage mode without this option.
+--query             Opt, the query string for selecting reads for coverage.
+--thread <Integer>  Opt, number of worker threads (yields n+1 total threads).
+--type              Req, the type of coverage to perform. Possible Values: [sequence,
+                      physical]. 
+--validation        Opt, how strict to be when reading a SAM or BAM. Possible values:
+                      [STRICT, LENIENT, SILENT].
+--version           Show version number.
 ~~~~
 
-For physical read depth:
+## Output
 
-~~~~{.text}
-java -jar qcoverage.jar -t phys \
-    --bam test1.bam --gff3 GRCh37.gff3 -o report.vcf --log logfile
-~~~~
+### standard mode output
 
+The default output is tab delimited text format, with or without option "--output-format TXT". Here file extension ".txt" is automatically append to putput file. 
```

Comment:
How about this:
"The default output is in tab delimited text format, which can be explicitly requested with the "--output-format TXT" option. Here the file extension ".txt" is automatically append to the output file."

Reply:
suggestion is taken.